scalableminds/webknossos
# thumbnails: correctly use zoom value if specified

Before, the thumbnail generation interpreted the zoom value as the exponent, but it was sent as the concrete value, we need a binary logarithm as conversion in between.

### Steps to test:
- specify zoom for a dataset (in edit dataset →view configuration)
- thumbnail should still work even for large zoom values
- thumbnail still displays a resolution 1:1 but it *selects* this resolution based on the zoom value of supplied (some zoom value changes will not have an impact on the thumbnail. If we want that, we need to change more here)

------
